modules/runner/src/integrationTest/java/org/apache/ignite/internal/sql/api/ItCommonApiTest.java

```diff
@@ -152,7 +152,6 @@ public void checkTimestampOperations() {
 
     /** Check transaction change status with erroneous statements. */
     @Test
-    // TODO should be removed after https://issues.apache.org/jira/browse/IGNITE-20534  is fixed.
     public void testTxStateChangedOnErroneousOp() {
```

Comment:
seems there was an intention to remove the test after the fix (but I still cant get why), and you just removed todo - what is correct here?

Reply:
It shuld to be removed of @lowka 's words.
I will do that.